parley/src/layout/mod.rs

```diff
@@ -280,6 +280,23 @@ pub struct Line<'a, B: Brush> {
     data: &'a LineData,
 }
 
+#[derive(Clone, Debug)]
```

Comment:
Let's get this back.

```suggestion
#[derive(Copy, Clone, Debug)]
```